### PR TITLE
Set up two-way sync for phone numbers

### PR DESCRIPTION
**Pull request by gcapalbo**
@benrudolph

### CODE REVIEW

**Comment by dimagimon** [comment]
There are 7 errors:

* corehq/apps/sms/tests/migration.py, line 650 - F841 local variable 'owner_doc_type' is assigned to but never used
* corehq/apps/sms/tests/migration.py, line 651 - F841 local variable 'owner_id' is assigned to but never used
* corehq/apps/sms/tests/migration.py, line 652 - F841 local variable 'phone_number' is assigned to but never used
* corehq/apps/sms/tests/migration.py, line 653 - F841 local variable 'backend_id' is assigned to but never used
* corehq/apps/sms/tests/migration.py, line 654 - F841 local variable 'ivr_backend_id' is assigned to but never used
* corehq/apps/sms/tests/migration.py, line 655 - F841 local variable 'verified' is assigned to but never used
* corehq/apps/sms/tests/migration.py, line 656 - F841 local variable 'contact_last_modified' is assigned to but never used

**Comment by gcapalbo** [comment]
(hold off on merge, need to fix the build)

File: corehq/apps/sms/tests/migration.py [hunk 1]
```diff
@@ -607,3 +619,134 @@ def testSQLSyncToCouch(self):
...
+        owner_doc_type = self.randomString()
+        owner_id = self.randomString()
+        phone_number = self.randomString()
+        backend_id = self.randomString()
+        ivr_backend_id = self.randomString()
+        verified = self.randomBoolean()
+        contact_last_modified = self.randomDateTime()
+
+    def testCouchSyncToSQL(self):
+        self.deleteAllLogs()
```
Comment (benrudolph) [inline corehq/apps/sms/tests/migration.py]: nit: could put in setUp

File: corehq/apps/sms/tests/migration.py [hunk 2]
```diff
@@ -607,3 +619,134 @@ def testSQLSyncToCouch(self):
...
+        self.assertEqual(self.getCouchCount(), 0)
+        self.assertEqual(self.getSQLCount(), 0)
+
+        # Test Create
+        couch_obj = VerifiedNumber()
+        self.setRandomCouchObjectValues(couch_obj)
+        couch_obj.save()
+
+        sleep(1)
```
Comment (benrudolph) [inline corehq/apps/sms/tests/migration.py]: is this an issue with a couch doc not being saved immediately or something?

**Comment by benrudolph** [comment]
minor comments, :+1: on :white_check_mark:

File: corehq/apps/sms/tests/migration.py (hunk 2 above)
Comment (gcapalbo) [inline corehq/apps/sms/tests/migration.py]: yeah i only see these issues locally, but for some reason if i don't wait the doc is not found

File: corehq/apps/sms/tests/migration.py (hunk 1 above)
Comment (gcapalbo) [inline corehq/apps/sms/tests/migration.py]: updated

**Pull request merged**